src/Stripe.net/Services/OAuth/OAuthAuthorizeUrlOptions.cs

```diff
@@ -0,0 +1,76 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class OAuthAuthorizeUrlOptions : BaseOptions
+    {
+        /// <summary>
+        /// Boolean to indicate that the user should always be asked to connect, even if they're
+        /// already connected. Defaults to <c>false</c>.
+        /// </summary>
+        [JsonProperty("always_prompt")]
+        public bool? AlwaysPrompt { get; set; }
+
+        /// <summary>
+        /// The unique identifier provided to your application, found in your
+        /// <see href="https://dashboard.stripe.com/account/applications/settings">application settings</see>.
+        /// </summary>
+        [JsonProperty("client_id")]
+        public string ClientId { get; set; } = StripeConfiguration.ClientId;
+
+        /// <summary>The only option at the moment is <c>code</c>.</summary>
```

Comment:
We usually do the comment on 3 lines.
also is `<c>....</c>` new?

Reply:
I sometimes put the comment on a single line if it fits under the 100 characters. Not a big deal one way or the other, it's rendered the same way.

`<c></c>` is used to inline code, like backquotes in Markdown: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/code-inline. It's already used in plenty of other comments.